admin_manual/enterprise_user_management/file_firewall.rst

```diff
@@ -0,0 +1,174 @@
+======================
+File Firewall(ES only)
+======================
+
+The File Firewall GUI enables you to create and manage firewall rule sets from 
+your ownCloud admin page. The File Firewall gives you finer-grained control of 
+access and sharing, with rules for allowing or denying access, and restrictions 
+per group, upload size, client devices, IP address, time of day, and many more 
+criteria. For additional flexibility the File Firewall also supports regular 
+expressions.
+
+Each rule consists of one or more conditions. A request matches a rule if 
+all conditions evaluate to true. If a request matches at least one of the 
+defined rules, the request is blocked.
+
+Administrators are excluded from the firewall when they 
+access the instance via the WebUI, to allow admins to modify the rules. 
+Otherwise they could potentially be locked out. Public share links and syncing 
+via WebDAV are subject to firewall rules for all users all the time.
+
+Figure 1 shows an empty firewall configuration panel. Set your logging level to 
+**Failures Only** for debugging, and create a new ruleset by clicking the **Add 
+Group** button. After setting up your rules you must click the **Save Rules** 
+button.
+
+.. figure:: images/firewall-1.png
+   :alt: Empty File Firewall configuration panel.
+   
+   *Figure 1: Empty File Firewall configuration panel*
+
+Figure 2 shows two rules. The first rule, **No Support outside 
+office hours**, prevents members of the support group from logging into the 
+ownCloud Web interface from 5pm-9am.
+
+The second rule prevents members of the qa-team group from accessing the Web UI 
+from IP addresses that are outside of the local network.
+
+.. figure:: images/firewall-2.png
+   :alt: Two example rules that restrict logins per user group.
+   
+   *Figure 2: Two example rules that restrict logins per user group*   
+
+All other users are not affected, and can log in anytime from anywhere.
+
+Available Conditions
+--------------------
+
+User Group
+ The user (is|is not) a member of the selected group.
+
+User Agent
+ The User-Agent of the request (matches|does not match) the given string.
+
+User Device
+  A shortcut for matching all known (`android`|`ios`|`desktop`) sync clients by 
+  their User Agent string.
+
+Request Time
+ The time of the request (has to|must not) in a single range from beginning 
+ time to end time.
+
+Request URL
+ The **full page URL** (has to|must not) (match|contain|begin with|end) with a 
```

Comment:
Seems like the last "with" could go inside the brackets to make "end with" in the brackets.